Source/Particles/Collision/CollisionHandler.H

```diff
@@ -23,7 +23,7 @@
 class CollisionHandler
 {
 public:
-    CollisionHandler ();
+    CollisionHandler (MultiParticleContainer* const mypc);
```

Comment:
Potentially, depending on the details in `BinaryCollision` and constructed classes within its constructor, this can also be passed as `MultiParticleContainer const * const mypc`

Reply:
Yes I think that makes sense to put it const as well here, I'll do that.